cmd/bosun/sched/bolt.go

```diff
@@ -103,6 +95,7 @@ func (s *Schedule) save() {
 		log.Printf("wrote %s: %v", name, conf.ByteSize(cw.written))
 		collect.Put("statefile.size", opentsdb.TagSet{"object": name}, cw.written)
 	}
+	s.Unlock()
```

Comment:
Don't want to defer becuase I don't want to include the bolt io time. Bolt does its own locking on transactions for that part. I'll add the unlock explicitly.